gencode/main.ml

```diff
@@ -100,7 +98,7 @@ let program =
   Cohttp_lwt_unix.Client.get
     ("http://www.unicode.org/emoji/charts/emoji-list.html"
      |> Uri.of_string) >>= fun (_, body) ->
-  Cohttp_lwt_body.to_string body >>= fun html ->
+  Cohttp_lwt__Body.to_string body >>= fun html ->
```

Comment:
PR says switch to dune but this is code changes. What happened in upstream cohttp ?

Reply:
I don't know. I've got latest cohttp installed, there wasn't any version constraint here so I just fixed it.

Reply:
Did you test it, that is, everything stayed the same yes? Even after the update of library

Reply:
I added a commit with a test which seems to work. You can run it with `dune exec test/test.exe`.